compose_files/docker-compose-forger.yml

```diff
@@ -45,21 +47,22 @@ services:
       - "RPC_PASSWORD=${ZEN_RPC_PASSWORD}"
       - "RPC_PORT=${ZEN_RPC_PORT}"
       - "SEED_TAR_GZ_URL=${ZEN_SEED_TAR_GZ_URL}"
+      - "USE_SEED_FILE=${ZEN_USE_SEED_FILE}"
+      - "FORCE_RESEED=${ZEN_FORCE_RESEED}"
     logging:
       driver: "json-file"
       options:
         max-size: "512m"
         max-file: "4"
 
   evmapp:
-    image: zencash/evmapp:${SC_COMMITTISH_EVMAPP}
-    container_name: evmapp
-    depends_on:
-      - zend
+    image: "zencash/evmapp:${EVMAPP_TAG}"
+    container_name: "${EVMAPP_CONTAINER_NAME:-evmapp}"
     restart: always
+    stop_grace_period: 10m
     networks:
       evmapp_network:
-        ipv4_address: ${SCNODE_IP_ADDRESS}
+        ipv4_address: ${EVMAPP_IP_ADDRESS}
```

Comment:
I think you should try use double quotes if works as expected here as well to follow your pattern before(all bash env vars enclosed in double quotes)